Lesson31/Gokhan/demo-app/src/shared-components/typography/Display/index.jsx

```diff
@@ -0,0 +1,14 @@
+import './styles.css';
+
+// export const Display = ({ color }) => {
+//     const className = () => `display ${color}`;
+//     return <title className={className()}>Display</title>
+// }
+
+export const Display = ({ color, children }) => {
+    return (
+        <div className={`display ${color}`}>
```

Comment:
Display element is going to be used as the most important heading therefore using `h1` element instead of div would be more appropriate.